unassigner/prepare_strain_data.py

```diff
@@ -63,7 +63,9 @@ def main(argv=None):
         action="store_true",
         help=("Remove all downloaded and processed files."),
     )
-    p.add_argument("--db-dir", help=("Filepath to download the files to."))
+    p.add_argument(
+        "--db-dir", default=".autobfx/", help=("Filepath to download the files to.")
```

Comment:
I think you mean `.unassigner/` rather than `.autobfx/`. Again, the database directory should go in the user's home directory, rather than the working directory.